dev-requirements.txt

```
@@ -4,6 +4,7 @@ flake8~=5.0
 flake8-bugbear~=22.9
 isort~=5.7
 pytest~=5.4.3
+coverage~=7.5.1
```

Should be able to loosen the requirement to
```suggestion
coverage~=7.5
```
I think.